src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2247,6 +2256,11 @@ private BoundExpression BindArgumentExpression(DiagnosticBag diagnostics, Expres
 
                     arguments[arg] = ((OutVarLocalPendingInference)argument).SetInferredType(parameterType, success: !hasErrors);
                 }
+                else if (argument.Kind == BoundKind.DefaultPendingInference)
+                {
+                    TypeSymbol parameterType = GetCorrespondingParameterType(ref result, parameters, arg);
+                    arguments[arg] = ((BoundDefaultPendingInference)argument).SetInferredType(parameterType);
+                }
                 else if (argument.Kind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
... because it should not be specified to work with only specific language contexts.

---
In reply to: [77467707](https://github.com/dotnet/roslyn/pull/13603#discussion_r77467707) [](ancestors = 77467707)